python/lsst/sdm/schemas/hsc.yaml

```diff
@@ -1072,6 +1072,11 @@ tables:
     datatype: boolean
     description: Interpolated pixel in the Source center. Measured on g-band.
     fits:tunit:
+  - name: g_pixelFlags_nodata
+    "@id": "#Object.g_pixelFlags_nodata"
+    datatype: boolean
+    description: Source is outside usable exposure region (masked NO_DATA). Measured on g-band.
+    fits:tunit:
```

Comment:
I am sorry but you will probably need to add blocks for all the nodataCenter flags here too